.buildkite/pipeline_perf.py

```diff
@@ -48,7 +48,7 @@ def build_group(test):
     test_path = test.pop("test_path")
     return group(
         label=test.pop("label"),
-        command=f"./tools/devtool -y test {devtool_opts} -- --nonci -s --dump-results-to-file --log-cli-level=INFO {test_path}",
+        command=f"./tools/devtool -y test {devtool_opts} -- --nonci --dump-results-to-file {test_path}",
```

Comment:
If we explicitly want no log outputs would it make sense to set `--log-cli-level=OFF`? ([`log::LevelFilter::Off`](https://docs.rs/log/latest/log/enum.LevelFilter.html)).